src/components/screens/editor/editor-screen.tsx

```diff
@@ -8,64 +8,60 @@ import Api from "api";
 import { NavigationTabContainer } from "styled/layouts/navigations";
 import NavigationTab from "components/layouts/navigations/navigation-tab";
 import { Metaform, MetaformVersion, MetaformVersionType } from "generated/client";
-import { AdminFormListStack, AdminFormTypographyField, VersionListHeader } from "styled/react-components/react-components";
-import { DataGrid, GridColDef } from "@mui/x-data-grid";
 import { NewUserButton } from "styled/layouts/admin-layout";
-import moment from "moment";
 import EditorScreenDrawer from "./editor-screen-drawer";
 import { useNavigate } from "react-router-dom";
-
-/**
- * Interface for single Metaform row
- */
-interface Row {
-  metaform: Metaform;
-  versions: MetaformVersion[];
-}
+import GenericLoaderWrapper from "components/generic/generic-loader";
+import EditorScreenTable from "./editor-table";
```

Comment:
Name could be more describing, what the component actually shows. List of Metaform versions?
Also, check folder hierarchy and move screen components away from screens-folder to dedicated folder, in this case editor-folder which already exists in this project.

Reply:
Done.